src/components/Timer/Timer.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+
+import styled from 'styled-components';
+
+const Timer = () => {    
+    const [minute, setMinute] = useState(0);
+    const [second, setSecond] = useState(0);
+    const [mill, setMill] = useState(0);
```

Comment:
I guess we should not keep this information in the state since we already have `setStopwatchTime`.

Reply:
Agreed.